ETBezierHandle.m

```diff
@@ -77,21 +81,25 @@ - (id) initWithActionHandler: (ETActionHandler *)anHandler
 		{
 			// FIXME: ugly
 			case NSCurveToBezierPathElement:
-				[handles addObject: AUTORELEASE([[ETBezierHandle alloc] initWithActionHandler: [ETBezierPointActionHandler sharedInstance]
-				                                                            manipulatedObject: aTarget
-																			         partcode: [path partcodeForControlPoint: 0 ofElement: i]])];
-				[handles addObject: AUTORELEASE([[ETBezierHandle alloc] initWithActionHandler: [ETBezierPointActionHandler sharedInstance]
-				                                                            manipulatedObject: aTarget
-																			         partcode: [path partcodeForControlPoint: 1 ofElement: i]])];
-				[handles addObject: AUTORELEASE([[ETBezierHandle alloc] initWithActionHandler: [ETBezierPointActionHandler sharedInstance]
-				                                                            manipulatedObject: aTarget
-																			         partcode: [path partcodeForControlPoint: 2 ofElement: i]])];
+                [handles addObject: AUTORELEASE([[ETBezierHandle alloc] initWithActionHandler: [ETBezierPointActionHandler sharedInstanceForObjectGraphContext: aContext]
```

Comment:
To reduce the length of these lines, I'd suggest to assign [ETBezierPointActionHandler sharedInstanceForObjectGraphContext: aContext] to a local variable and passing it as an argument